### DISCUSSION TITLE
Addressables Loading

### DISCUSSION THREAD

**Discussion by Skipper0X** [Cysharp/UniTask#481] [Q&A]
most of the code is using async properly but
i need to Wait the UniTask while using in an old system, old system is not supporting async execution at the moment,
and i'm stuck at this point, do you have any suggestion,
converting the UniTask to Task to call Wait() freezes the Unity

**Comment by LukasDavidSchwarz** [comment]
I want to do the same thing, but I'm still stuck on converting UniTask to Task. Could you tell me how you did it? And did you get it working without freezing?
